client/landing/stepper/hooks/use-get-site-suggestions-query.ts

```diff
@@ -32,6 +32,7 @@ export const useGetSiteSuggestionsQuery = ( {
 		refetchOnWindowFocus,
 		enabled,
 		onSuccess,
+		queryKey: [],
```

Comment:
Even though `[]` is a valid key, we should provide a meaningful key. Let's change this:

```suggestion
		queryKey: [ 'site-suggestions' ],
```